pkgs/desktops/lxqt/lxqt-config/default.nix

```diff
@@ -44,6 +45,7 @@ mkDerivation rec {
     libkscreen
     liblxqt
     libqtxdg
+    xkeyboard_config
```

Comment:
If this is only used in postPatch this does not need to be listed here.

Reply:
The file `${xkeyboard_config}/share/X11/xkb/rules/base.lst` should be present at runtime. Therefore the package which provides it should be a runtime dependence, right?

Reply:
Since there is a reference in the binary itself there is no need to be declared directly in `buildInputs`, AFAIK.

Reply:
I am not sure. I do not think that referencing a file from a given package in the installed binary will automatically make that package a runtime dependence. At least I have never heard or read about this feature. Is it documented somewhere?

Reply:
It should work because as long Nix is concerned, since as long there is a reference to the file Nix itself will not remove the reference (even after a garbage collection).

I don't know if there is a documentation for it. I remember reading it in Nix Pills, but not exactly what chapter.

Reply:
Does it work when there is a mention of the dependence path in the nix store in any installed file, or only with binaries?

Reply:
Any dependency, Nix doesn't have the concept of what is a binary file or not.

Reply:
Ok. I have removed `xkeyboard_config` from `buildInputs`.